ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -970,7 +977,14 @@
               fail: true
               timeout: 135
               type: absolute
-          publishers: []
+          publishers: 
+          - archive:
+              allow-empty: true
+              artifacts: '*sonobuoy*.tar.gz'
+              case-sensitive: true
+              default-excludes: true
+              fingerprint: false
+              only-if-success: false
```

Comment:
can use like below to check success
- flexible-publish:
        publishers:
        - condition:
            condition-kind: 'Current status'
            condition:
              success: false

Reply:
We can introduce `flexible-publish` if it can provide new features we need (We can create a separate PR for the potential upgrade because it could impact more jobs). Otherwise I feel the current plugin already meet our requirements.